packages/cli/src/ceramic-daemon.ts

```diff
@@ -489,6 +495,31 @@ export class CeramicDaemon {
     })
   }
 
+  async getModelsFromIndex(req: Request, res: Response): Promise<void> {
```

Comment:
`getIndexedModels`?  `getModelsFromIndex` sounds like you're querying the index itself rather than the configuration about what **will be** indexed